### DISCUSSION TITLE
Github page link to .md file not showing in Jekyll theme

### DISCUSSION THREAD

**Discussion by Kartibok** [community/community#22083] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,</p>
<p>my basic index page <a href="https://kartibok.github.io/Capture-the-Flag/" rel="nofollow noopener">CTF</a> works fine and shows the Jekyll Theme.</p>
<p>My issue is that my link in the page goes back to github and another .md file. Why is this not showing the same Jekyll theme. Am I wrong to assume that this would be automatic? Do I need to do something to each .md page to make it fit in with the same theme?</p>
<p>All help appreciated.</p>
<p>Regards</p>
<p>K</p>

**Comment by otter-computer** [comment, marked as answer]
<p>This is happening because you’ve created a direct link to your file inside your repository on <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a>. For example, this is one of the links you’ve created inside your <code>index.md</code> file:</p>
<pre><code class="lang-auto">* [Basic Penetration CTF](https://github.com/Kartibok/Capture-the-Flag/blob/d53cf932e14ce638530e90a157d02879d97aa224/basic_pentesting.md)
</code></pre>
<p>You can see the rendered version of this page on your Pages site here: <a href="https://kartibok.github.io/Capture-the-Flag/basic_pentesting" rel="nofollow noopener">https://kartibok.github.io/Capture-the-Flag/basic_pentesting</a></p>
<p>In order to link this page instead of the markdown file inside your repository you’ll need to create a relative link to the file:</p>
<pre><code class="lang-auto">* [Basic Penetration CTF](basic_pentesting.md)
</code></pre>
<p>When your site is built Pages will transform this link to point to the rendered HTML version instead of linking to the markdown version in your repository. This will allow the link to be clickable inside your repository and <em>also</em> take you to the correct rendered version on your Pages site.</p>

**Comment by Kartibok** [comment]
<p>Thomas,</p>
<p>Thank you very much. Such a simple solution!!</p>
<p>Your help has been appreciated.</p>
<p>Regards</p>
<p>K</p>
